### PR TITLE
Update lt.m3u

### PR DESCRIPTION
**Pull request by PapasBlessas**
Added following channels:
LNK, BTV, LRTTV (LRT HD), LRT Plius, 2TV, InfoTV, TV1.

Non-LRT channels don't work when everything after ".m3u8" is deleted.

### CODE REVIEW

**Comment by PapasBlessas** [comment]
The non-LRT channels don't work if everything after .m3u8 is deleted

File: streams/lt.m3u [hunk 1]
```diff
@@ -19,3 +19,17 @@ https://radijas.lrt.lt/radijas/master.m3u8
 https://radijas.lrt.lt/klasika/master.m3u8
 #EXTINF:-1 tvg-id="LRTOpus.lt",LRT Opus (1080p)
 https://radijas.lrt.lt/opus/master.m3u8
+#EXTINF:-1 tvg-id="LNK.lt",LNK (1080p)
+https://live.lnk.lt/lnk_live/smil:lnk_live.smil/playlist.m3u8?tokenstarttime=0&tokenendtime=1683259096&tokenhash=iOBCxbO8M6HzvpKYFaBxWNZcCdDiYfMk2sVLDu_Jx6MHIvOIOUESXwJRGYHau7Q3j70CiTayc3axjxix4WSktA==
+#EXTINF:-1 tvg-id="BTV.lt",BTV (1080p)
+https://live.lnk.lt/lnk_live/smil:btv_live.smil/playlist.m3u8?tokenstarttime=0&tokenendtime=1683259326&tokenhash=Z-qCE9n27ryLgi3p8uaIqwObuY5Eym1DHVoIpfmGKW6VjN-sjqsiboIh34wlyt3RVe49Vrx64giMMAj0TCJqeQ==
+#EXTINF:-1 tvg-id="LRTTV.lt",LRT HD (1080p)
+https://lrthd.lrt.lt/secure/lrt_hd/TK8DdmrYi1ahTMKCxVOdFg/1683237332/master.m3u8
```
Comment (freearhey) [inline streams/lt.m3u:27]: offline
image

File: streams/lt.m3u [hunk 2]
```diff
@@ -19,3 +19,17 @@ https://radijas.lrt.lt/radijas/master.m3u8
...
+#EXTINF:-1 tvg-id="LRTPlius.lt",LRT Plius (1080p)
+https://plius.lrt.lt/secure/plius/0bxGmNnqOX1sQp9NbmVL6Q/1683237472/master.m3u8
```
Comment (freearhey) [inline streams/lt.m3u:29]: offline
image

**Review by freearhey — CHANGES_REQUESTED** [review]

File: streams/lt.m3u (hunk 1 above)
Comment (PapasBlessas) [reply streams/lt.m3u:27]: image
Works for me in VLC. Is it possible it's geo-blocked, but showing up as offline in the test? If so, I'll add the geo-blocked label.

File: streams/lt.m3u (hunk 2 above)
Comment (PapasBlessas) [reply streams/lt.m3u:29]: >Works for me in VLC. Is it possible it's geo-blocked, but showing up as offline in the test? If so, I'll add the geo-blocked label.

same with this playlist

File: streams/lt.m3u (hunk 1 above)
Comment (freearhey) [reply streams/lt.m3u:27]: Perhaps you are right, the links are indeed geo-blocked.

**Comment by kaikyoofers** [comment]
Seems like the links are tokened? They stop working after a while

**Comment by PapasBlessas** [comment]
> Seems like the links are tokened, They stop working after a while

Is there any way around it? Removing the token info from the link makes the playlist unplayable

File: streams/lt.m3u (hunk 1 above)
Comment (BellezaEmporium) [reply streams/lt.m3u:27]: https://www.lrt.lt/servisai/stream_url/live/get_live_url.php?channel=LTV1 and works from a French IP.

You can scrape the link from there.

File: streams/lt.m3u (hunk 2 above)
Comment (BellezaEmporium) [reply streams/lt.m3u:29]: Same here : https://www.lrt.lt/servisai/stream_url/live/get_live_url.php?channel=LTV2

File: streams/lt.m3u (hunk 1 above)
Comment (PapasBlessas) [reply streams/lt.m3u:27]: Tried the link in Streamtest, returns offline. My guess is that it's either geo-blocked to europe or the test just can't see it

**Comment by PapasBlessas** [comment]
Seems like all of my added links are tokened. I'm closing the pull request as there is nothing else I can add or really do...

**Pull request closed without merging**